pkg/subctl/cmd/cloud/utils/utils.go

```diff
@@ -0,0 +1,33 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+Copyright Contributors to the Submariner project.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"fmt"
+	"github.com/submariner-io/submariner-operator/internal/exit"
+)
+
+const (
+	InfraIDFlag   = "infra-id"
+	RegionFlag    = "region"
+	ProjectIDFlag = "project-id"
+)
+
+// ExpectFlag exits with an error if the flag value is empty.
+func ExpectFlag(flag, value string) {
+	if value == "" {
+		exit.WithMessage(fmt.Sprintf("You must specify the %v flag", flag))
+	}
+}
```

Comment:
Why did you add this file? `ExpectFlag` is already defined [here](https://github.com/submariner-io/submariner-operator/blob/0c4ef6c9942718116c8ca46983e03e079c000dbb/pkg/subctl/cmd/utils/utils.go#L63) and the constants are already defined elsewhere.